inst/Description.qml

```diff
@@ -33,6 +33,11 @@ Description
 		title:	qsTr("Logistic Regression")
 		func:	"RegressionLogistic"
 	}
+    Analysis
+    {
+        title: qsTr("Generalized Linear Model")
+        func: "GeneralizedLinearModel"
+    }
```

Comment:
Use tabs instead of spaces in Qml files

Reply:
I do use tabs (the tab button on the keyboard) but it always comes out spaces. Any idea why?

Reply:
And just out of curiosity, how did you find out that it was spaces instead of tabs?

Reply:
Yes, it depends on the settings of your text editor that you use to edit the qml files. If you do that in R Studio, then it's likely it automatically replaces tabs with spaces (which is what we want for R files). I usually edit qml files in Qt creator or VS Code where you can setup tab formatting specifically for qml files...

> And just out of curiosity, how did you find out that it was spaces instead of tabs?

Well, you can set up your text editor to highlight different types of whitespaces (usually it's under Preferences/Display/Show whitespace characters)

But I saw it on GitHub directly because I use the following extension https://github.com/refined-github/refined-github which nicely highlights the whitespaces in the browser.

Reply:
Thanks! Super cool.